apps/backend/src/libs/modules/cloud-formation/cloud-formation.module.ts

```diff
@@ -0,0 +1,121 @@
+import {
+	CloudFormationClient,
+	CreateStackCommand,
+	DeleteStackCommand,
+	DescribeStacksCommand,
+	waitUntilStackCreateComplete,
+	waitUntilStackDeleteComplete,
+} from "@aws-sdk/client-cloudformation";
+
+import { type MeetingService } from "~/modules/meetings/meeting.service.js";
+
+import { type Logger } from "../logger/logger.js";
+import {
+	Capability,
+	ExceptionMessage,
+	OutputKey,
+	ParameterKey,
+	StackPrefix,
+} from "./libs/enums/enums.js";
+
+type Constructor = {
+	credentials: {
+		accessKeyId: string;
+		secretAccessKey: string;
+	};
+	imageId: string;
+	logger: Logger;
+	meetingService: MeetingService;
+	region: string;
+	templateBody: string;
```

Comment:
Let's pass templateBody to the create method directly to have a possibility to use different templates if required